packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -73,9 +73,10 @@ const editStateReducer = (state, action) => {
   }
 };
 
-const toPriceRequestKey = (identifier, time) => time + "," + identifier;
-const toVotingAccountAndPriceRequestKey = (votingAccount, identifier, time) =>
-  votingAccount + "," + time + "," + identifier;
+// toPriceRequestKey( identifier,time, ancillarydata )
+const toPriceRequestKey = (...args) => args.join(",");
+// toVotingAccountAndPriceRequestKey( votingAccount, identifier, time, ancillarydata )
+const toVotingAccountAndPriceRequestKey = (...args) => args.join(",");
```

Comment:
`toPriceRequestKey` and `toVotingAccountAndPriceRequestKey` now implement the same function. I find it a bit more readable to define the different vars that will be joined with commas, similar to how you define it in the comments. Why'd you change it?

Reply:
Yea good point, i was a bit on the fence about this. The reason why is that it can accept arbitrary arguments now, for instance with or without ancillary data if needed, though this might not be a good enough reason.  I dont mind changing it back.

Reply:
> The reason why is that it can accept arbitrary arguments now, for instance with or without ancillary data if needed

This is definitely useful, but I think its confusing to have two functions implement the same logic. So if you want the flexibility of an unspecified amount of args, maybe consolidate the functions into one `toPriceRequestKey`